<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationDialog</class>
 <widget class="QDialog" name="TransformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformation Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/Registar.qrc">
    <normaloff>:/images/bunny.png</normaloff>:/images/bunny.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Random</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>max_angle</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="maxAngleLineEdit">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>max_distance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="maxDistanceLineEdit">
           <property name="text">
            <string>0.005</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>57</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QCheckBox" name="rotateAroundCentroidCheckBox">
           <property name="text">
            <string>Rotate around Centroid</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Normalize</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="sendCentroidOriginCheckBox">
           <property name="text">
            <string>Send Centroid to Origin</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="alignPCAXYZCheckBox">
           <property name="text">
            <string>Align PCA axis to XYZ axis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="scaleUnitCheckBox">
           <property name="text">
            <string>Scale size of Bounding Box to 1.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="uniformRadioButton">
       <property name="text">
        <string>Uniform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="separateRadioButton">
       <property name="text">
        <string>Separate</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="defaultButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>tabWidget</zorder>
  <zorder></zorder>
  <zorder>rotateAroundCentroidCheckBox</zorder>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <resources>
  <include location="../res/Registar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>TransformationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
